### PR TITLE
Add docker container for Progress DB

### PR DESCRIPTION
**Pull request by mmarkauskas**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM gitlab.baltic-amadeus.lt:5005/progress/progress-docker/progress-build:12.6.0-centos
```
Comment (PauliusKu) [inline docker/Dockerfile:1]: I think we can use PROGRESS_BASE_IMAGE variable here

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Docker
+Docker part of app: database and backend (Progress OpenEdge).
```
Comment (PauliusKu) [inline docker/README.md:2]: Docker container with sports2020 DB for testing.

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+services:
+  app-backend:
```
Comment (PauliusKu) [inline docker/docker-compose.yml:3]: rename to `sports-db-126`

later we'll have 122 and 117 containers too.

File: docker/Dockerfile (hunk 1 above)
Comment (mmarkauskas) [reply docker/Dockerfile:1]: Pretty sure we can't, .env is not directly accessible within the dockerfile. The approach probably would be to pass it as a build argument? Idk

Comment (PauliusKu) [reply docker/Dockerfile:1]: Something like that:
`FROM ${DB_NAME}`
and then add PROGRESS_BASE_IMAGE to environment variables in docker compose file

**Review by PauliusKu — APPROVED** [review]

**Review by Einingis — APPROVED** [review]

**Pull request merged**